sale_confirm_activities_mandatory/models/mail_activity_type.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class MailActivityType(models.Model):
+    _inherit = "mail.activity.type"
+
+    category = fields.Selection(selection_add=[('validation', "Validation")])
+    validator_group_ids = fields.Many2many(
+        "res.groups",
+        "mail_activity_groups_rel",
+        "activity_id",
+        "group_id",
+    )
```

Comment:
All this mechanism deserves its own module (maybe in OCA/social?) as it could be used by any flow I suppose (not only sale)?

Reply:
+1